### ISSUE TITLE
No way to export the scan results

### ISSUE THREAD

**Issue by sudhaharv** [subgraph/Vega#103]
Once the scanning is done, we need to raise bugs. for that we have to export the test results.

we could not find any options to export the scan results

**Comment by iBringsTheRuckus** [comment]
Would be great to import too.

**Comment by ouaziz** [comment]
i need to export results, how can i do ?????

**Comment by dma** [comment]
Closing, duplicate of #13. We will implement this (and possibly importing).

**Comment by rdhnsop** [comment]
Is there an ETA on when we will be able to export results?
